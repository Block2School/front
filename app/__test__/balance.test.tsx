import renderer from 'react-test-renderer'
import React from 'react'
import Balance from '../components/balance/balance'
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import { shallow } from 'enzyme';
import { render, screen } from '@testing-library/react';
import { useWeb3React } from "@web3-react/core";



describe('<Balance/>', () => {
  let library: any;
  let account: any;
  let wrapper: any;
  test('should render correctly with no argument', () => {
    wrapper = shallow(<Balance account={undefined} library={undefined} />)
  });

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render correctly with argument', () => {
    wrapper = shallow(<Balance account={account} library={library} />)
  });
})