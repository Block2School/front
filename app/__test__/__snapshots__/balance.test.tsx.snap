// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Balance/> should match snapshot 1`] = `ShallowWrapper {}`;
